app/assets/javascripts/search.js

```diff
@@ -28,6 +28,5 @@
 
 $(function(){
```

Comment:
Does this serve a purpose? Removing the `console.warn` is valid but does it make sense to leave the rest of it?